api/v1/configurationpolicy_types.go

```diff
@@ -65,10 +66,40 @@ type Condition struct {
 	Message string `json:"message,omitempty" protobuf:"bytes,5,opt,name=message"`
 }
 
-// Target defines the list of namespaces to include/exclude
-type Target struct {
+// namespaceSelector defines the list of namespaces to include/exclude for objects defined in
+// spec.objectTemplates. All selector rules are ANDed.
+type NamespaceSelector struct {
+	// include is an array of filepath expressions to include namespaces by name.
```

Comment:
Could you also document that if it's empty but a label selector is specified, this acts as being set to `*`?

Reply:
Ah, good point. Yes, I'll add it.